index.bs

```diff
@@ -1827,49 +1827,93 @@ To <dfn for="WebTransportError">set up</dfn> a {{WebTransportError}} |error| wit
 
 This section describes the [[QUIC]] protocol behavior of methods defined
-in this specification, utilizing [[WEB-TRANSPORT-HTTP3]].
+in this specification, utilizing [[WEB-TRANSPORT-HTTP3]]. Cause and effect may
+not be immediate due to buffering.
 
   <table class="data">
     <colgroup class="header"><col></colgroup>
     <colgroup><col></colgroup>
     <thead>
       <tr>
-        <th>Method</th>
+        <th>API Method</th>
         <th>QUIC Protocol Action</th>
       </tr>
     </thead>
     <tbody>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.{{WritableStream/abort}}(errorCode)</td>
-        <td>send RESET_STREAM with errorCode</td>
+        <td>[=stream/Reset|sends RESET_STREAM=] with errorCode</td>
       </tr>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.{{WritableStream/close}}()</td>
-        <td>send STREAM_FINAL</td>
+        <td>[=stream/Send|sends=] STREAM_FINAL</td>
       </tr>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.getWriter().{{WritableStreamDefaultWriter/write}}()</td>
-        <td>send STREAM</td>
+        <td>[=stream/Send|sends=] STREAM</td>
       </tr>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.getWriter().{{WritableStreamDefaultWriter/close}}()</td>
-        <td>send STREAM_FINAL</td>
+        <td>[=stream/Send|sends=] STREAM_FINAL</td>
       </tr>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.getWriter().{{WritableStreamDefaultWriter/abort}}(errorCode)</td>
-        <td>send RESET_STREAM with errorCode</td>
+        <td>[=stream/Reset|sends RESET_STREAM=] with errorCode</td>
       </tr>
       <tr>
         <td>{{WebTransportBidirectionalStream/readable}}.{{ReadableStream/cancel}}(errorCode)</td>
-        <td>send STOP_SENDING with errorCode</td>
+        <td>[=send STOP_SENDING|sends STOP_SENDING=] with errorCode</td>
       </tr>
       <tr>
-        <td>{{WebTransportBidirectionalStream/readable}}.getReader().{{ReadableStreamDefaultReader/read}}()</td>
-        <td>receive STREAM or STREAM_FINAL</td>
+        <td>{{WebTransportBidirectionalStream/readable}}.getReader().{{ReadableStreamGenericReader/cancel}}(errorCode)</td>
+        <td>[=send STOP_SENDING|sends STOP_SENDING=] with errorCode</td>
       </tr>
       <tr>
-        <td>{{WebTransportBidirectionalStream/readable}}.getReader().{{ReadableStreamGenericReader/cancel}}(errorCode)</td>
-        <td>send STOP_SENDING with errorCode</td>
+        <td>wt.{{WebTransport/close}}(closeInfo)</td>
+        <td>[=session/terminate|terminates=] connection with closeInfo<br>
+      </tr>
+    </tbody>
+  </table>
+
+  <table class="data">
+    <colgroup class="header"><col></colgroup>
+    <colgroup><col></colgroup>
+    <thead>
+      <tr>
+        <th>QUIC Protocol Action</th>
+        <th>API Effect</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>[=stream/Receive|received=] STOP_SENDING with errorCode</td>
```

Comment:
Is this really something you want to tie to streams receiving? When we talk about receiving frames, the thing that does that is the connection.